src/notifications/entities/notification.entity.ts

```
@@ -0,0 +1,57 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  Index,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+import {Tenant} from '../../tenants/entities/tenant.entity'; 
```

ESLint

[error] A space is required after '{'.

(object-curly-spacing)